allocations/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+import json
+import logging
+from django.conf import settings
+from datetime import datetime
+
+logger = logging.getLogger('allocations')
+
+class Allocation(models.Model):
+    STATUS = (
+        ('inactive', 'expired or overused'),
```

Comment:
Does the second item here map to a UI element somehow? I presume it's not stored in the DB?

Reply:
no. it's not mapped to UI element and not stored in the db. but django `choices` requires two items.